generate-docs/tools/API Coverage Report.csv

```diff
@@ -2094,34 +2094,34 @@ ExcelScript.PivotAxis,filter,EnumField,Poor,false
 ExcelScript.PivotAxis,row,EnumField,Poor,false
 ExcelScript.PivotAxis,unknown,EnumField,Poor,false
 ExcelScript.PivotDateFilter,N/A,Class,Good,false
-ExcelScript.PivotDateFilter,comparator,Method,Good,false
-ExcelScript.PivotDateFilter,condition,Method,Fine,false
-ExcelScript.PivotDateFilter,exclusive,Method,Good,false
-ExcelScript.PivotDateFilter,lowerBound,Method,Fine,false
-ExcelScript.PivotDateFilter,upperBound,Method,Fine,false
-ExcelScript.PivotDateFilter,wholeDays,Method,Fine,false
+ExcelScript.PivotDateFilter,comparator,Property,Good,false
+ExcelScript.PivotDateFilter,condition,Property,Fine,false
+ExcelScript.PivotDateFilter,exclusive,Property,Good,false
+ExcelScript.PivotDateFilter,lowerBound,Property,Fine,false
+ExcelScript.PivotDateFilter,upperBound,Property,Fine,false
+ExcelScript.PivotDateFilter,wholeDays,Property,Fine,false
 ExcelScript.PivotField,N/A,Class,Poor,false
-ExcelScript.PivotField,applyFilter(filter),Method,Good,true
-ExcelScript.PivotField,clearAllFilters(),Method,Good,false
-ExcelScript.PivotField,clearFilter(filterType),Method,Fine,false
+ExcelScript.PivotField,applyFilter(filter),Method,Fine,false
+ExcelScript.PivotField,clearAllFilters(),Method,Poor,false
+ExcelScript.PivotField,clearFilter(filterType),Method,Poor,false
 ExcelScript.PivotField,getFilters(),Method,Poor,false
 ExcelScript.PivotField,getId(),Method,Poor,false
 ExcelScript.PivotField,getItems(),Method,Poor,false
 ExcelScript.PivotField,getName(),Method,Poor,false
-ExcelScript.PivotField,getPivotItem(name),Method,Good,false
+ExcelScript.PivotField,getPivotItem(name),Method,Poor,false
 ExcelScript.PivotField,getShowAllItems(),Method,Poor,false
 ExcelScript.PivotField,getSubtotals(),Method,Poor,false
-ExcelScript.PivotField,isFiltered(filterType),Method,Fine,false
+ExcelScript.PivotField,isFiltered(filterType),Method,Poor,false
 ExcelScript.PivotField,setName(name),Method,Poor,false
 ExcelScript.PivotField,setShowAllItems(showAllItems),Method,Poor,false
 ExcelScript.PivotField,setSubtotals(subtotals),Method,Poor,false
-ExcelScript.PivotField,sortByLabels(sortBy),Method,Good,false
-ExcelScript.PivotField,sortByValues(sortBy,Method,Good,false
+ExcelScript.PivotField,sortByLabels(sortBy),Method,Poor,false
+ExcelScript.PivotField,sortByValues(sortBy,Method,Poor,false
 ExcelScript.PivotFilters,N/A,Class,Fine,false
-ExcelScript.PivotFilters,dateFilter,Method,Good,false
-ExcelScript.PivotFilters,labelFilter,Method,Good,false
-ExcelScript.PivotFilters,manualFilter,Method,Good,true
-ExcelScript.PivotFilters,valueFilter,Method,Good,true
+ExcelScript.PivotFilters,dateFilter,Property,Good,false
+ExcelScript.PivotFilters,labelFilter,Property,Good,false
+ExcelScript.PivotFilters,manualFilter,Property,Good,false
+ExcelScript.PivotFilters,valueFilter,Property,Good,false
```

Comment:
Same issue for these... ([ExcelScript.PivotFilters.manualFilter](https://docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.pivotfilters?view=office-scripts#excelscript-excelscript-pivotfilters-manualfilter-member), ExcelScript.PivotFilters.valueFilter)

I only spot checked so there may be other instances